pom.xml

```diff
@@ -120,5 +120,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <!--
+            * [JENKINS-51876] Updated core, Java version, and dependencies
+            * [JENKINS-4758] Remove old JDK installer from the plugin
```

Comment:
I guess this should have been JENKINS-47586.

Reply:
yep, is a typo, in any case, you have the description of the breaking changes at the changelog https://github.com/jenkinsci/ssh-slaves-plugin/blob/master/CHANGELOG.md#127-aug-26-2018